site/docs/foundations/data-visualization/chart-accessibility.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Chart accessibility
+layout: DetailTechnical
+sidebar:
+  priority: 5
+---
+
+Charts rely heavily on visuals to help users understand complex data, so it’s important to ensure that your charts meet all accessibility requirements. Use color appropriately, include visual cues, provide text for assistive technologies, and ensure any interactions are accessible.
+
+## Categorical colors
+
+### Color contrast
+
+Users with visual impairments should be able to fully comprehend data sets that are represented by different colors. To cater for a broad spectrum of visual impairments, you must use the categorical colors which are WCAG 4.5:1 compliant.
+
+If you use custom colors or Hex codes, they may not meet non-text contrast requirements so you will be responsible to ensure that the resulting contrast is compliant. By using Salt tokens correctly, you will achieve at least 3:1 contrast between the background and other non-text chart elements such as bars, lines, segments and icons.
+
+<Image
+  src="/img/foundations/dataviz-hex.png"
+  srcDark="/img/foundations/dataviz-hex-dark.png"
+  alt="Diagram of best practices using categorical color versus color picking."
+/>
+
+### Color order
+
+Apply the categorical colors in the recommended order. All categorical colors have similar contrast levels, as they use the 500 series swatch from their respective color ramps.
+
+<Image
+  src="/img/foundations/dataviz-color-order.png"
+  srcDark="/img/foundations/dataviz-color-order-dark.png"
+  alt="Diagram of the categorical color palette."
+/>
+
+A well-defined, distinct color palette ensures that everyone can easily interpret the data. Using the set categorical palette colors minimizes the risk of misinterpretation and ensures a visual order. Don’t use shades of the same color.
+
+<Image
+  src="/img/foundations/dataviz-monochromatic.png"
+  srcDark="/img/foundations/dataviz-monochromatic-dark.png"
+  alt="Diagram of best practices using categorical palette versus monochromatic palette."
+/>
+
+You can enhance your chart’s accessibility by adopting additional visual strategies other than color such as:
+
+- Pattern overlays for different categories
+- Direct labelling where possible
+- Adding a border with at least a 3:1 contrast ratio to distinguish between segments
```

Comment:
We should say what the 3:1 contrast ratio should be against e.g., the background of the container, the segment backgrounds or all of the above.